src/components/Cards/styled.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+interface Props {
+  isLoading: boolean;
+}
+
+export const CardsWrapper = styled.div<Props>`
+  align-self: ${(p) => (p.isLoading ? 'center' : 'flex-start')};
```

Comment:
p - is a bad name. destructuring will eat more symbols, but this is common way of syntax in stc.
```suggestion
  align-self: ${({isLoading}) => isLoading ? 'center' : 'flex-start'};
```

Reply:
right, done